initfiles/componentfiles/configxml/esp.xsl

```diff
@@ -68,6 +68,7 @@
                 <xsl:copy-of select="/Environment/Software/Directories"/>
             </Software>
             <xsl:copy-of select="/Environment/Hardware/cost"/>
+            <xsl:copy-of select="/Environment/Software/locale"/>
```

Comment:
Could this be worked out at runtime?

Reply:
esp.xml configuration is generated from the environment.xml when the esp service is started with hpcc-init.  There is a mechanism in ESP that supports the reloading of an updated configuration (into a running esp) and, so the locale can be changed and reloaded at runtime.  Is that what you mean?

Reply:
I was thinking more of reading the locale from the machine / OS instance, so the human is not involved?  But given they have to enter the pricing anyway its fine as is.

Reply:
I see what you mean.  I'd imagine the locale would need to be installed in the OS to be able to do that and that's a bit cumbersome in the containerized deployment.